packages/dev/core/src/Shaders/ShadersInclude/prePassDeclaration.fx

```diff
@@ -2,7 +2,9 @@
 #extension GL_EXT_draw_buffers : require
 layout(location = 0) out highp vec4 glFragData[{X}];
 highp vec4 gl_FragColor;
-
+#ifdef PREPASS_LOCAL_POSITION
```

Comment:
we need to make sure that their webgpu version are also implemented

Reply:
Done